core/config_utils_test.go

```diff
@@ -25,6 +25,11 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	// To enable debug logging during test execution, set this to "LevelDebug"
+	configUtilsTestLogLevel LogLevel = LevelDebug
```

Comment:
Shouldn't this be configured to error level?

Reply:
yes... yes it should :joy:

Reply:
fixed it